rewards/boost/calc_boost.py

```diff
@@ -87,7 +88,7 @@ def badger_boost(current_block: int, chain: str):
 
     for addr, boost in badger_boost.items():
         boost_metadata = boost_info.get(addr, {})
-        boost_data[addr] = {
+        boost_data[Web3.toChecksumAddress(addr)] = {
```

Comment:
Where is the source coming from on this?  To match further down, we should be checksumming the addr before saving in the `badger_boost` var

Reply:
This comes from the subgraph queries - these should be checksummed instead